<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="MainWidget">
   <widget class="QPushButton" name="PasswordBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Give password</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="PasswordText">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>36</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Your strong and secure password</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="EncryptMenu">
    <property name="title">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KEncryptMenu">
    <property name="title">
     <string>Key Encrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="KDecryptMenu">
    <property name="title">
     <string>Key Decrypt</string>
    </property>
   </widget>
   <widget class="QMenu" name="PasswordMenu">
    <property name="title">
     <string>Strong password</string>
    </property>
   </widget>
   <addaction name="EncryptMenu"/>
   <addaction name="KEncryptMenu"/>
   <addaction name="KDecryptMenu"/>
   <addaction name="PasswordMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
